data/modules/general/general.yml

```diff
@@ -1,6 +1,6 @@
 category: General
 description: Guidance that all projects need to follow.
-minimumRisk: Low Risk
+minimumRisk: Mandatory
```

Comment:
This is good :-). At first glance it makes you go "wait, what?" but makes sense.